e2e/atlas/helper_test.go

```diff
@@ -240,6 +242,17 @@ func RandProjectNameWithPrefix(prefix string) (string, error) {
 	return prefixedName, nil
 }
 
+func MongoDBMajorVersion() (string, error) {
+	atlasClient := mongodbatlas.NewClient(nil)
+	atlasClient.BaseURL, _ = url.Parse("https://cloud-dev.mongodb.com/")
```

Comment:
the endpoint returns 4.4 in production

Reply:
please use `MCLI_OPS_MANAGER_URL` for this, don't hard code it